src/MainParallel.java

```diff
@@ -495,6 +499,38 @@ private static void processCard(String cardName, AtomicInteger doneCounter) thro
         doneCounter.incrementAndGet();
     }
 
+    private static String getArchetype(Document dom){
+        Element cardtableCategories = dom.getElementsByClass("cardtable-categories").first();
+        if(cardtableCategories == null) return "";
+
+        String archetype = "";
+        List<String> templist = new ArrayList<>(5); //I think 5 is good enough for an initial capacity
+
+        /*things to note:
+        - "Archetypes and series" will always come first if there is anything concerning archetypes
+         */
+        for(Element hlist : cardtableCategories.getElementsByClass("hlist")){
+            Element dl = hlist.getElementsByTag("dl").first();
+            String dt = dl.getElementsByTag("dt").first().outerHtml();
+            Elements archetypes = null;
+
+            //check if the left side contains the words archetypes
+            if(dt.contains("Archetype") || dt.contains("Archetypes") || dt.contains("archetypes")) archetypes = dl.getElementsByTag("dd");
```

Comment:
- `dt.toLowerCase().contains("archetype")` is enough
 - Use brackets for `if`
 - Revert the condition:

```
    if(!dt.toLowerCase().contains("archetype") ) {
        continue;
    }

    archetypes = dl.getElementsByTag("dd");
```